tox.ini

```diff
@@ -15,9 +15,14 @@ commands =
 
 [testenv:black]
 deps =
-    black>=22.3.0,<23
+    black>=23.1.0,<24
 commands = black --check .
 
+[testenv:isort]
+deps =
+    isort>=5.12.0,<5.13
+commands = isort --profile black --check src tests
+
```

Comment:
While we're at it ; can we add flake8 as well ?

```ini
[testenv:flake8]
deps =
    flake8>=6.0.0,<7.0
commands = flake8 src --count --max-line-length=88 --statistics --extend-ignore=E203
```

Reply:
Do We need use `flake8` to check tests?

Reply:
It's not testing the tests, it's testing `src/`